### ISSUE TITLE
2.16 --> 1.x version switching is buggy

### ISSUE THREAD

**Issue by jenweber** [ember-learn/ember-api-docs#386]
Navigating from 2.16 to 1.3, 1.4, 1.5, or 1.6 results in a 404. It's our good friend undefined.json once again. I discovered it while trying to add some more tests around versioning and rendering.

80% chance that I'm responsible. Debugging 😞 

Also seems to affect 2.2...?

The adapter seems to be making requests to `ember-1.6.1-ember-RSVP` for example, instead of `ember-1.6.1-ember` like non-failing requests

**Comment by jenweber** [comment]
Traced to RSVP showing up under Packages for 1.2 - 1.6

Could patch in getModel of module.js. But I don't think it's supposed to be there.

**Comment by toddjordan** [comment]
yeah, this is a bug in ember-jsonapi-docs  will put in a fix this week.

**Comment by MinThaMie** [comment]
It's so old :D
